rgw/v2/tests/s3_swift/configs/test_lc_rule_multipart_transition_notifications.yaml

```diff
@@ -0,0 +1,41 @@
+# test_bucket_lifecycle_object_expiration_transition.py
+# bz: https://bugzilla.redhat.com/show_bug.cgi?id=2166576
```

Comment:
Please also add the testcase ID.

Reply:
added polarion id @viduship 
please re-review